src/VirtoCommerce.Platform.Core/Security/IUserPasswordHasher.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.AspNetCore.Identity;
+
+namespace VirtoCommerce.Platform.Core.Security
+{
+    /// <summary>
+    /// Basic interface for platform password hashers
+    /// </summary>
+    public interface IUserPasswordHasher
```

Comment:
Why do not inherit directly from `IPasswordHasher<ApplicationUser>`?

Reply:
Ok.